routing_model/index.html

```diff
@@ -531,6 +531,55 @@ <h2>Process for Adult/Elder promotion and demotion including merge</h2>
      We have to be careful that the section follows up on relocation once merged, so we may want to avoid active relocation when merging.
   </p>
 
+  <button class="collapsible">CheckElderTimeout</button>
+  <div class="content">
+    <p>As for incrementing work units, we want to update the eldership status of all nodes in a section on a syncrhonised, regular basis.<br/>
+    For this reason, it makes sense to have a timer going through Parsec.<br/>
+    Note that this timer has to be relatively fast so that it remains highly unlikely that 1/3 of the elders in any section would go offline within one timer's duration.
+    </p>
+  </div>
+  <button class="collapsible">Merge RPC</button>
+  <div class="content">
+    <p>The merge RPC is the RPC a section sends to their neighbour section when they need merging.<br/>
```

Comment:
A section sends a Merge RPC to their neighbour section when they are ready to merge at the given SectionInfo digest.